apps/ledger-live-mobile/src/components/WebPlatformPlayer/index.tsx

```diff
@@ -1,697 +1,19 @@
-      requestAccount,
-      receiveOnAccount,
-      signTransaction,
-      broadcastTransaction,
-      startExchange,
-      completeExchange,
-      signMessage,
-    ],
-  );
-  const handleSend = useCallback((request: JSONRPCRequest): Promise<void> => {
-    targetRef?.current?.postMessage(JSON.stringify(request));
-
-    return Promise.resolve();
-  }, []);
-  const [receive] = useJSONRPCServer(handlers, handleSend);
-  const handleMessage = useCallback(
-    e => {
-      // FIXME: event isn't the same on desktop & mobile
-      // if (e.isTrusted && e.origin === manifest.url.origin && e.data) {
-      if (e.nativeEvent?.data) {
-        receive(JSON.parse(e.nativeEvent.data));
-      }
-    },
-    [receive],
-  );
-
-  const handleLoad = useCallback(() => {
-    if (!widgetLoaded) {
-      tracking.platformLoadSuccess(manifest);
-      setWidgetLoaded(true);
-    }
-  }, [manifest, widgetLoaded]);
-
-  const handleReload = useCallback(() => {
-    tracking.platformReload(manifest);
-    setLoadDate(new Date());
-    setWidgetLoaded(false);
-  }, [manifest]);
-
-  const handleError = useCallback(() => {
-    tracking.platformLoadFail(manifest);
-  }, [manifest]);
-
-  useEffect(() => {
-    navigation.setOptions({
-      headerRight: () => (
-        <View style={styles.headerRight}>
-          <ReloadButton onReload={handleReload} loading={!widgetLoaded} />
-          <InfoPanelButton
-            loading={!widgetLoaded}
-            setIsInfoPanelOpened={setIsInfoPanelOpened}
-          />
-        </View>
-      ),
-    });
-  }, [navigation, widgetLoaded, handleReload, isInfoPanelOpened]);
-  useEffect(() => {
-    tracking.platformLoad(manifest);
-  }, [manifest]);
-  return (
-    <SafeAreaView style={[styles.root]}>
-      <InfoPanel
-        name={manifest.name}
-        icon={manifest.icon}
-        url={manifest.homepageUrl}
-        uri={uri.toString()}
-        description={manifest.content.description}
-        isOpened={isInfoPanelOpened}
-        setIsOpened={setIsInfoPanelOpened}
-      />
-      <WebView
-        ref={targetRef}
-        startInLoadingState={true}
-        showsHorizontalScrollIndicator={false}
-        showsVerticalScrollIndicator={false}
-        renderLoading={() => (
-          <View style={styles.center}>
-            <ActivityIndicator size="large" />
-          </View>
-        )}
-        originWhitelist={manifest.domains}
-        allowsInlineMediaPlayback
-        source={{
-          uri: uri.toString(),
-        }}
-        onLoad={handleLoad}
-        onMessage={handleMessage}
-        onError={handleError}
-        overScrollMode="content"
-        bounces={false}
-        mediaPlaybackRequiresUserAction={false}
-        automaticallyAdjustContentInsets={false}
-        scrollEnabled={true}
-        style={styles.webview}
-      />
-    </SafeAreaView>
-  );
+const WebViewWrapper = ({ manifest, inputs }: Props) => {
+  if (semver.satisfies("2.0.0", manifest.apiVersion)) {
```

Comment:
```suggestion
  if (matchVersion({ version: "2.0.0" }, manifest)) {
```

Shouldn't we be more consistent with desktop implementation and use `matchVersion` from `@ledgerhq/live-common/platform/filters`?

Even better, couldn't we refacto this component that seems very similar with the one in desktop into a single component hosted in LLC (maybe using [React HOC](https://reactjs.org/docs/higher-order-components.html))